### DISCUSSION TITLE
Add entry_count variable inside {exp:channel:categories} tag

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#2936] [Feature Requests]
Currently there's no way to output the number of open entries inside a {exp:channel:categories} tag, so using it to output a list of product categories with an indication of how many entries are in that category is not possible.

Could we add the entry_count as a variable, something like:

```
{exp:channel:categories show_empty="no" category_group="1" style="linear"}
  <li><a href="{route='products/category'}">{category_name} ({entry_count})</a></li>
{/exp:channel:categories}
```

**Comment by intoeetive** [comment]
Just want to add that I like this feature request, however in order to implement it we'll likely need to add some extra database tables and manipulations.

If we calculate the stats live on each page load, that might create too heavy load on large websites. If we store this information however (like we have `total_entries` for channel) we need to consider open/closed entries, entry dates, channels etc - because all of those can be requested separately
